Channels/Operation/01_ManageOffers.md

```diff
@@ -0,0 +1,339 @@
+        - ![Radiobutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *semiautomatic, changes must be confirmed by another user*   
+          Changes in a PIM product are automatically applied to the offer after confirmation by another user. The initial offer is only applied after confirmation by another user.    
+        - ![Radiobutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *automatic*   
+          Changes in a PIM product are automatically applied to the offer. The initial offer is also automatically applied.
+
+    + Select the appropriate initial status for the offer in the *Initial offer status* section:   
+        - ![Radiobutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *Active*   
+            The offer is active. It is uploaded to the marketplace and activated directly. Thus, it is also displayed in the marketplace, where it can be sold immediately.   
+        - ![Radiobutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *Inactive*   
+            The offer is inactive. It is uploaded to the marketplace, but still remains inactive. Thus, it is not displayed in the marketplace and cannot yet be sold.   
+        - ![Radiobutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *Offline*   
+            The offer is created in *Omni-Channel*, but not yet uploaded to the marketplace. Thus, it is unknown to the marketplace and cannot be sold there.
+
+9. Click the [CREATE OFFERS] button.   
+    The offer creation has been triggered. The *Product import from PIM scheduled* pop-up window is displayed.
+
+    ![Import scheduled](../../Assets/Screenshots/Channels/Offers/Offers/ProductImportScheduled.png "[Import scheduled]")
+
+> [Info]  As the creation process starts asynchronously, it may take some time until the offer creation has been started and all offers have been created. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner to update the list of offers.   
+Switch to *Offer Import > Tab PIM->CHANNELS* to check the queue for all scheduled offer imports from *PIM*, see [Check the scheduled offer imports](#check-the-scheduled-offer-imports).
+
+
+
+### Create a manual offer
+
+Create an offer manually if either the Actindo *PIM* module is not installed or you want to independently create an offer.
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections#create-a-connection).
+
+#### Procedure
+
+*Omni-Channel > Offers > Tab OFFERS*
+
+![Offers](../../Assets/Screenshots/Channels/Offers/Offers/Offers.png "[Offers]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The [Manual offer ![Manual offer](../../Assets/Icons/Document.png "[Manual offer]") ] button and the [Create from PIM product ![Create from PIM product](../../Assets/Icons/Package.png "[Create from PIM product]") ] button are displayed.
```

Comment:
If the PIM (or PimChannelsConnection) is not installed at all, the "Create from pim product" option is not available at all. Therefor the "Create manual offer" window is opened immediately after clicking the plus icon in the grid.